newIDE/app/src/InstancesEditor/index.js

```diff
@@ -540,7 +541,7 @@ export default class InstancesEditor extends Component<Props> {
   setZoomFactor = (zoomFactor: number) => {
     this.props.onChangeInstancesEditorSettings({
       ...this.props.instancesEditorSettings,
-      zoomFactor: Math.max(Math.min(zoomFactor, 10), 0.01),
+      zoomFactor: Math.max(Math.min(zoomFactor, 100), 0.01),
```

Comment:
Nitpick: it could be 128 and 1/128 as powers of 2 are used.

Reply:
I have not stored the zoom factor as a power of 2 as you suggested. it's still the result of 2**(x/8)

Reply:
I don't think it's needed at the moment. It could be an improvement for the future

Reply:
Yes, it's probably better the way you did it, but it will still do a x2 every 8 steps (with some rounding errors that probably don't matter) so it won't go to x100 but it will go about x128. So if we reach the limit, we can still find some integer zooms from x128 but probably not from x100.
It's not a very common use case though.